programs/rodeos/streams/rabbitmq.hpp

```diff
@@ -41,14 +42,6 @@ class rabbitmq : public stream_handler {
        , queue_name_( std::move( queue_name))
    {
       ilog("Connecting to RabbitMQ address ${a} - Queue: ${q}...", ("a", address)( "q", queue_name_));
-      std::atomic<bool> error = false;
-      eosio::amqp_handler declare_queue( address_, fc::seconds(60), fc::milliseconds(200), [&error](const std::string& err){
-         elog("AMQP Queue error: ${e}", ("e", err));
-         appbase::app().quit();
-         error = true;
-      } );
-      if( !error ) declare_queue.declare_queue(queue_name_);
-      if( error ) return;
```

Comment:
I guess we don't currently have any tests that needed this. In the future any streaming test will need to create the queue manually.